pxtlib/docsrender.ts

```diff
@@ -282,10 +282,18 @@ namespace pxt.docs {
 
         // Add print button
         const printBtnHtml = `
-            <button id="printbtn" class="circular ui icon right floated button hideprint" title="${lf("Print this page")}">
+        <div id="printbtn" class="circular ui icon top right floated pointing dropdown button hideprint" title="${lf("Print this page")}">
```

Comment:
maybe we can move this HTML into its own .html file under/docfiles; then this change would not require a cloud update.

Reply:
Yes that could work. I'll add a new commit with this

Reply:
Quick question about this @pelikhan, the lf("Color") and lf("Black and White"). If I put this html under docfiles. How should I render this localizable label? I think I could export a function from runner.ts ? Or what would be a better approach? I don't know exactly when the lf() function can be called. Thanks!